core/lib/types/src/contract_verification_api.rs

```diff
@@ -157,7 +157,7 @@ pub enum CompilerType {
 pub enum CompilerVersions {
     #[serde(rename_all = "camelCase")]
     Solc {
-        compiler_zksolc_version: String,
+        compiler_zksolc_version: String, // FIXME: optional?
```

Comment:
Logically, it would make sense to not specify `compiler_zksolc_version` for contracts with EVM bytecode. This requires a DB migration etc., so I'd prefer to implement it in a follow-up PR.